doc/tutorials/run_reformatting_script_with_docker.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+Running the reformat-all script requires to have multiple dependencies installed. To overcome this problem, instead of trying to install all the necessary dependencies on your own,
```

Comment:
```suggestion
Running the reformat-all script requires multiple dependencies. To overcome this problem, instead of trying to install all the necessary dependencies on your own,
```